optuna_dashboard/ts/action.ts

```diff
@@ -49,6 +50,25 @@ export const actionCreator = () => {
     setStudyDetailState(studyId, newStudy)
   }
 
+  const setTrialState = (
```

Comment:
What do you think? To better describe its behavior.
```suggestion
  const setTrialStateValues = (
```